cli/cmd/health.go

```diff
@@ -42,28 +45,33 @@ func runHealth(cmd *cobra.Command, args []string) {
 		"args":    args,
 	})
 
-	kubeconfig, err := getKubeconfig()
-	if err != nil {
-		contextLogger.Fatalf("Error in finding kubeconfig file: %s", err)
+	if debug {
+		log.SetLevel(log.DebugLevel)
 	}
 
-	cs, err := k8sutil.NewClientset(kubeconfig)
-	if err != nil {
-		contextLogger.Fatalf("Error in creating setting up Kubernetes client: %q", err)
-	}
-
-	p, diags := getConfiguredPlatform()
+	lokoConfig, diags := getLokoConfig()
 	if diags.HasErrors() {
 		for _, diagnostic := range diags {
 			contextLogger.Error(diagnostic.Error())
 		}
+
 		contextLogger.Fatal("Errors found while loading cluster configuration")
 	}
 
-	if p == nil {
-		contextLogger.Fatal("No cluster configured")
+	kubeconfig, err := getKubeconfig(contextLogger, lokoConfig, true)
+	if err != nil {
+		contextLogger.Debugf("Error in finding kubeconfig file: %s", err)
+		contextLogger.Fatal("Suitable kubeconfig file not found. Did you run 'lokoctl cluster apply' ?")
 	}
 
+	cs, err := k8sutil.NewClientset(kubeconfig)
+	if err != nil {
+		contextLogger.Fatalf("Error in creating setting up Kubernetes client: %q", err)
```

Comment:
Seems little confusing.

```suggestion
		contextLogger.Fatalf("Error in setting up Kubernetes client: %q", err)
```

Reply:
Out of scope of this PR, but I fixed it in separate commit.